parameter.schema.json

```diff
@@ -158,6 +178,40 @@
       },
       "required": ["value", "base"]
     },
+    "distribution_chi_2": {
+      "type": "object",
+      "properties": {
+        "df": {
+          "description": "The degrees of freedom",
+          "$ref": "./content-types.schema.json#/definitions/latex_expression"
+        }
+      },
+      "required": ["df"]
+    },
+    "distribution_normal": {
+      "type": "object",
+      "properties": {
+        "mu": {
+          "description": "The mean of the distribution (default 0)",
+          "$ref": "./content-types.schema.json#/definitions/latex_expression"
+        },
+        "sigma": {
+          "description": "The standard deviation of the distribution (default 1)",
+          "$ref": "./content-types.schema.json#/definitions/latex_expression"
+        }
+      },
+      "required": ["df"]
```

Comment:
@Graspleric this should be `mu` and `sigma` righ?

Reply:
@screambeard You are absolutely correct, fixed it now.